ui/test/nuxeo-date.test.js

```diff
@@ -0,0 +1,87 @@
+/**
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import { html, fixture } from '@nuxeo/testing-helpers';
+import moment from '@nuxeo/moment';
+import '../widgets/nuxeo-date.js';
+
+function setNuxeoConfigDateFormat(format, value) {
+  window.Nuxeo = window.Nuxeo || {};
+  window.Nuxeo.UI = window.Nuxeo.UI || {};
+  window.Nuxeo.UI.config = window.Nuxeo.UI.config || {};
+  window.Nuxeo.UI.config[format] = window.Nuxeo.UI.config[format] || value;
+}
+
+suite('nuxeo-date', async () => {
+  test('Should hide the nuxeo-tooltip if provided format and tooltipFormat are equal', async () => {
```

Comment:
for consistency sake, we use in our test description the `should ... when` formulation.
```suggestion
  test('Should hide the nuxeo-tooltip when provided format and tooltipFormat are equal', async () => {
```